classes/class2/README.md

```diff
@@ -1 +1,303 @@
-#Class 2
+#JavaScript
+
+## JavaScript is Everywhere
+
+### Atwood's Law
+	
+>Any application that can be written in JavaScript, will eventually be written in JavaScript.
+
+[Jeff Atwood](http://blog.codinghorror.com/the-principle-of-least-power/)
+
+## The Language
+
+Luckily, you're probably already familiar with a language similar to JavaScript. Things like variable assignment,function definition, and control flow aren't much different from Python. We'll get into the differences later.
+
+Type `node` in your command line to bring up the Node JavaScript REPL (read-evaluate-print-loop).
```

Comment:
This is a good point -- it might be a little cumbersome to explain the developer console in the Readme, though. I think this is better mentioned in class; node in the command line is really simple (and means they have to have node installed).